all-the-icons.el

```diff
@@ -1070,10 +1070,10 @@ When PFX is non-nil, ignore the prompt and just install"
                        ((member system-type '(gnu gnu/linux gnu/kfreebsd))
                         (concat (or (getenv "XDG_DATA_HOME")
                                     (concat (getenv "HOME") "/.local/share"))
-                                "/fonts/"))
+                                "/fonts/all-the-icons/"))
```

Comment:
This is not what I had in mind when I said "customize" lol. See if you can extract this to a variable.

Reply:
Damnit. I see what you mean. Something like `defcustom` and so forth ?

Reply:
Ok, I've had another go at it. Let me know if this works.